mangaki/mangaki/utils/mal.py

```diff
@@ -168,6 +170,12 @@ def list_works_from_a_user(self,
                 continue
             try:
                 title = entry.find('series_title').text
+                # Take all non-empty synonyms.
+                synonyms = [
```

Comment:
I'd agree, if we'd use a generator expression rather than a list, but you're right, I prefer it also.